yewprint/src/html_elements.rs

```diff
@@ -1,22 +1,31 @@
 use yew::prelude::*;
-use yewtil::{Pure, PureComponent};
 
 macro_rules! build_component {
     ($name:ident, $props_name:ident, $tag:tt, $class:literal) => {
-        pub type $name = Pure<$props_name>;
+        // pub type $name = dyn Component<$props_name, Message = $props_name, Properties = $props_name>;
 
-        #[derive(Debug, Clone, PartialEq, Properties)]
+        #[derive(Debug, Clone, PartialEq, yew::prelude::Properties)]
```

Comment:
You already imported the prelude.

```suggestion
        #[derive(Debug, Clone, PartialEq, Properties)]
```

Reply:
I was wondering about this. I wanted to qualify it since in the previous version, (or at least in Yewtils) it referenced a different location of Properties, which was now private. I thought qualifying it would solve it, but it didn't, and it ended up getting committed.

Reply:
If it was in the lib it should be accessible from the crate itself (or by using the root import `::`). But in this case it's in the doc so as long as it compiles it doesn't matter. The shorter the better